Mathlib/LinearAlgebra/Determinant.lean

```diff
@@ -336,6 +336,27 @@ theorem bot_lt_ker_of_det_eq_zero {𝕜 : Type*} [Field 𝕜] [Module 𝕜 M] {f
 lemma det_mulLeft (a : R) : (mulLeft R a).det = a := by simp
 lemma det_mulRight (a : R) : (mulRight R a).det = a := by simp
 
+theorem pi_det [Module.Free R M] [Module.Finite R M] (f : ι → M →ₗ[R] M) :
```

Comment:
```suggestion
theorem det_pi [Module.Free R M] [Module.Finite R M] (f : ι → M →ₗ[R] M) :
```
If you think about what this means (and ignore that you have written it with dot notation), you see why I recommend this order.

Reply:
Yes, the name made no sense.